src/kirby/components/empty-state/empty-state.component.tns.html

```diff
@@ -0,0 +1,10 @@
+<FlexboxLayout flexDirection="column" justifyContent="space-around">
+  <StackLayout class="box" alignSelf="center">
+    <kirby-icon *ngIf="iconName" [name]="iconName" size="lg" outline="true" class="icon"></kirby-icon>
+    <Label *ngIf="title" class="kirby-text-normal-bold title" [text]="title"></Label>
+    <Label *ngIf="text" class="kirby-text-normal message" [text]="text" textWrap="true"></Label>
```

Comment:
Use `.p` class as kirby-text-normal will be deprecated:
```suggestion
    <Label *ngIf="text" class="p message" [text]="text" textWrap="true"></Label>
```

Reply:
`kirby-text-normal` is used instead of `p`, because `p` adds a `margin-bottom`. If we use a `p` we need to zero out the margin-bottom.

Reply:
Fixed